packages/w/waline.json

```
@@ -20,10 +20,11 @@
       {
         "basePath": "dist",
         "files": [
-          "*.js"
+          "waline.css",
+          "waline.js",
+          "pageview.js"
         ]
       }
     ]
-  },
-  "filename": "Waline.min.js"
+  }
```

```suggestion
  },
    "filename": "waline.js"
```
We can still keep the filename as the recommended file for folks to use